src/python/turicreate/test/test_image_classifier.py

```diff
@@ -330,6 +352,29 @@ def test_evaluate_explore(self):
         evaluation.explore()
 
 
+class ImageClassifierSqueezeNetTest(ImageClassifierTest):
+    @classmethod
+    def setUpClass(self):
+        super(ImageClassifierSqueezeNetTest, self).setUpClass(
+            model="resnet-50",
+            input_image_shape=(3, 227, 227),
+            tol=0.005,
+            num_examples=200,
+            feature="2048",
+        )
+
+
+class ImageClassifierSqueezeNetTest(ImageClassifierTest):
```

Comment:
I'm a bit confused by the changes in the rest of this file. We already have classes named `ImageClassifierSqueezeNetTest` and `VisionFeaturePrintSceneTest`. Why are you defining them?

Reply:
Yeah, this was very stupid of me. 
I wanted to create 3 more classes that would inherit properties of `ImageClassifierTest` and would use deep features to create model and then do the same test but instead of giving them different names I just ended up giving them same names. I will correct it and change it accordingly and put it in me next PR.